components/create_post/create_post.jsx

```diff
@@ -536,21 +544,48 @@ class CreatePost extends React.PureComponent {
         const {
             currentChannel: updateChannel,
             userIsOutOfOffice,
+            allowReferencedGroups,
+            channelMemberCountsByGroup,
+            currentChannelMembersCount
         } = this.props;
 
-        const currentMembersCount = this.props.currentChannelMembersCount;
         const notificationsToChannel = this.props.enableConfirmNotificationsToChannel && this.props.useChannelMentions;
+        let memberCount = 0;
+        let timezoneCount = 0;
+        let groups = [];
+        if (this.props.enableConfirmNotificationsToChannel && !containsAtChannel(this.state.message)) {
+            // Groups mentioned in users text
+            groups = groupsMentionedInText(this.state.message, allowReferencedGroups);
+            if (groups.length > 0) {
+                const groupsWith5MembersInChannel = groups.map((group) => channelMemberCountsByGroup[group.id]).filter((group) => group);
```

Comment:
is `groupsWith5MembersInChannel` the correct name?

Reply:
I have changed the filter to better align with the name.